caffe2/onnx/backend.cc

```diff
@@ -1082,7 +1082,7 @@ Caffe2Ops Caffe2Backend::CreateDynamicSlice(
   // arguments to the caffe2 Slice operator.
   std::string axes_tensor;
   if (onnx_node->node.input_size() > 2) {
-    axes_tensor = onnx_node->node.input(3);
+    axes_tensor = onnx_node->node.input(2);
```

Comment:
The DynamicSlice spec has 3-4 inputs, with the 4th optional input as ```axes```. So maybe the fix should be on the if condition
 ```if (onnx_node->node.input_size() > 3) {```

Reply:
Oh, yes, DynamicSlice operator schema in ONNX has 3-4 inputs, the 4th is axes.
So I submit another commit, Is that OK?